feedparser/mixin.py

```diff
@@ -193,6 +193,7 @@ def __init__(self):
         self.svgOK = 0
         self.title_depth = -1
         self.depth = 0
+        self.hasContent = 0
```

Comment:
any reason for it not being:
* bool (True/False)
* snake case?

Reply:
Not that I can see, other than to be consistent with the other state variables (infeed, inentry, etc.) All lowercase would be more consistent (I guess `svgOK` and `title_depth` are the odd ones).

Reply:
I'll accept it as-is. There is a _lot_ of legacy code and formatting in feedparser (which is ~17 years old now!), and it'll take some work to get that standardized.